applications/nrf_desktop/doc/usb_state.rst

```diff
@@ -46,7 +46,8 @@ Boot protocol configuration
 If the device is meant to support the boot protocol, set the following options:
 
 #. Enable :kconfig:`CONFIG_USB_HID_BOOT_PROTOCOL`.
-#. Make sure :kconfig:`CONFIG_USB_HID_PROTOCOL_CODE` is set to either the mouse or the keyboard code.
+#. Currently dongle can only support either HID boot keyboard or HID boot mouse reports. To use dongle for forwarding HID boot keyboard reports set
```

Comment:
Not dongle but USB state in general (the module can also be used for peripherals).
Also I am not sure if we need to inform about values here, since the Kconfig option comes from Zephyr and it's documented here.